projects/experimental/directives/button-group-card/button-group-card.style.less

```diff
@@ -0,0 +1,52 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+[tuiButtonGroupCard] {
+    display: flex;
+    min-width: 21.4375rem;
+    border-radius: 1.5rem;
+
+    &[data-surface='dark'] > [tuiButton][tuiAppearance] {
+        color: #fff;
+
+        &:hover {
+            background: rgba(256, 256, 256, 0.13);
+        }
+    }
+
+    > [tuiButton][tuiButtonVertical] {
+        flex-grow: 1;
+        flex-basis: 0;
+        border-radius: 0;
+        padding: 1.125rem 0.25rem;
+        min-width: 6.625rem;
+        gap: 0.5rem;
+    }
+
+    > [tuiButton][tuiButtonVertical]:only-child {
+        flex-direction: row;
+        border-radius: 0.75rem;
+        padding: 1rem 0;
+    }
+
+    > [tuiButton]:first-child {
+        border-top-left-radius: 1.5rem;
+        border-bottom-left-radius: 1.5rem;
+    }
+
+    > [tuiButton]:last-child {
+        border-top-right-radius: 1.5rem;
+        border-bottom-right-radius: 1.5rem;
+    }
+
+    &:has([tuiButton]:only-child) {
+        border-radius: 0.75rem;
+    }
+
+    &:nth-child(2) > [tuiButton][tuiButtonVertical]:first-child {
+        padding-left: 1.75rem;
+    }
+
+    &:nth-child(2) > [tuiButton][tuiButtonVertical]:last-child {
```

Comment:
This seems wrong, we should probably just have padding around buttons and `justify-content: space-between` instead of manually adjusting paddings for children.

Reply:
<img width="722" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/6b59d37c-4c16-4869-919b-7102acade299">

but the space should be clickable